spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/GravitinoSparkConfig.java

```diff
@@ -13,5 +13,19 @@ public class GravitinoSparkConfig {
   public static final String GRAVITINO_HIVE_METASTORE_URI = "metastore.uris";
   public static final String SPARK_HIVE_METASTORE_URI = "hive.metastore.uris";
 
+  public static final String LAKEHOUSE_ICEBERG_CATALOG_BACKEND = "catalog-backend";
```

Comment:
after #2620 is merged, we could use the properties defined in `IcebergCatalogPropertiesMetadata`

Reply:
Could we change this later in other pr? cc @FANNG1

Reply:
sure, just a notice